src/neural/blas/se_unit.h

```diff
@@ -0,0 +1,58 @@
+/*
+ This file is part of Leela Chess Zero.
+ Copyright (C) 2018 The LCZero Authors
+
+ Leela Chess is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Leela Chess is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <cstddef>
+
+namespace lczero {
+
+// Squeeze-excitation unit
+class SEUnit {
+ public:
+  SEUnit() = delete;
+
+  // Batched forward inference.
+  static void Forward(const size_t batch_size, const size_t channels,
```

Comment:
What's the point of having a class with a single static function? Why not make it just a free-standing function (and to put what's now in private section into anonymous namespace of .cc file)